crates/forge-runner/src/lib.rs

```diff
@@ -0,0 +1,594 @@
+use crate::fuzzer::RandomFuzzer;
+use crate::running::blocking_run_from_test;
+use crate::test_case_summary::TestCaseSummary;
+use crate::test_crate_summary::TestCrateSummary;
+use anyhow::{anyhow, Context, Result};
+use cairo_felt::Felt252;
+use cairo_lang_runner::SierraCasmRunner;
+use cairo_lang_sierra::ids::ConcreteTypeId;
+use cairo_lang_sierra::program::{Function, Program};
+use cairo_lang_sierra_to_casm::metadata::MetadataComputationConfig;
+use cairo_lang_utils::ordered_hash_map::OrderedHashMap;
+use camino::Utf8PathBuf;
+use console::style;
+use futures::stream::FuturesUnordered;
+use futures::StreamExt;
+use once_cell::sync::Lazy;
+use scarb_artifacts::StarknetContractArtifacts;
+use smol_str::SmolStr;
+use starknet::core::types::BlockId;
+use std::collections::HashMap;
+use std::sync::Arc;
+use test_collector::FuzzerConfig;
+use test_collector::TestCase as CollectedTestCase;
+use test_collector::{ForkConfig as ForkConfigTrait, LinkedLibrary, RawForkParams};
```

Comment:
Those aliases make it unreadable, please remove them. If you want to rename them do it in separate PR

Reply:
It's either these aliases or introducing weird names to structs defined in this file